test/speculatelog_test.c

```diff
@@ -1,8 +1,14 @@
 #include <mtev_defines.h>
 #include <mtev_log.h>
 #include <stdio.h>
+
+#define UNUSED(x) (void)(x)
```

Comment:
What's the reason for all of these "unused" functions? I'm assuming this is some weird compilation requirement?